packages/bot-web-ui/src/components/dashboard/bot-builder/toolbar/toolbar.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { Icon, Dialog } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import LoadModal from 'Components/load-modal';
+import SaveModal from 'Components/save-modal';
+import { tabs_title } from 'Constants/bot-contents';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+import ToolbarButton from './toolbar-button';
+import WorkspaceGroup from './workspace-group';
+
+type TToolbar = {
+    active_tab: string;
+    file_name: string;
+    has_redo_stack: boolean;
+    has_undo_stack: boolean;
+    is_dialog_open: boolean;
+    is_drawer_open: boolean;
+    is_running: boolean;
+    is_stop_button_disabled: boolean;
+    is_stop_button_visible: boolean;
+    closeResetDialog: () => void;
+    onOkButtonClick: () => void;
+    onResetClick: () => void;
+    onRunButtonClick: () => void;
+    onSortClick: () => void;
+    onUndoClick: () => void;
+    onZoomInOutClick: () => void;
+    toggleSaveLoadModal: () => void;
+    is_mobile: boolean;
+    toggleStrategyModal: () => void;
+    toggleLoadModal: () => void;
+    toggleSaveModal: () => void;
+};
+
+const Toolbar = (props: TToolbar) => {
+    const {
+        is_mobile,
+        is_running,
+        active_tab,
+        is_dialog_open,
+        onOkButtonClick,
+        closeResetDialog,
+        toggleStrategyModal,
+        toggleLoadModal,
+        toggleSaveModal,
+    } = props;
+
+    return (
+        <>
+            {is_mobile ? (
+                <div className='toolbar'>
+                    <div className='toolbar__section'>
+                        <ToolbarButton
+                            button_id='db-toolbar__import-button--mobile'
+                            button_classname='toolbar__btn--icon'
+                            buttonOnClick={toggleLoadModal}
+                            icon={<Icon icon='IcFolderOpenFilled' color='active' size={16} />}
+                            button_text={localize('Load')}
+                        />
+                        <ToolbarButton
+                            button_id='db-toolbar__quick-strategy-button--mobile'
+                            button_classname='toolbar__btn--icon'
+                            buttonOnClick={toggleStrategyModal}
+                            icon={<Icon icon='IcPuzzle' color='active' size={16} />}
```

Comment:
Size 16 is the default value. pls check the component implementation.
```suggestion
                            icon={<Icon icon='IcPuzzle' color='active' />}
```

Reply:
yes the default size if 16 ✅